src/components/Login.js

```diff
@@ -40,123 +40,117 @@ const themeColors = {
   4: ['#FF94DB', 'white'],
 };
 
-class Login extends React.Component {
-  state = {
-    dummy: false,
-    index: Math.floor(Math.random() * 5),
-  };
+function Login {
```

Comment:
Since we are changing Login from a class to a function, we need to have a proper function declaration.
Try changing this line to `function Login(props) {`. We need the parentheses to indicate that it's a function, and we need a `props` parameter inside in order to access any props passed down to the function, such as `props.initialState` in line 114.

Side note: since the only thing that we are exporting in this file is the Login function, instead of writing `export default Login;` at the bottom, we can remove that line and instead export it at the top of the function declaration by stating `export default function Login(props) {` on this line. But this is not necessary!